### PR TITLE
Fix the CI environment (fedmsg / fedora-messaging)

### PR DESCRIPTION
**Pull request by cverna**
Signed-off-by: Clement Verna <cverna@tutanota.com>

### CODE REVIEW

**Comment by cverna** [comment]
jenkies test

File: devel/ci/integration/resultsdb/Dockerfile [hunk 1]
```diff
@@ -1,22 +1,17 @@
-FROM centos:7
```
Comment (bowlofeggs) [inline devel/ci/integration/resultsdb/Dockerfile:1]: Is this change related to the waiverdb issue?

Comment (cverna) [reply devel/ci/integration/resultsdb/Dockerfile:1]: We don't have fedora-messaging in proper EPEL and we run waiverdb on f29 in prod (https://pagure.io/waiverdb/blob/master/f/Dockerfile#_1).

Comment (bowlofeggs) [reply devel/ci/integration/resultsdb/Dockerfile:1]: @cverna yes, but this is the resultsdb container, which I would expect to run on EPEL in production (not actually sure, just an expectation).

**Comment by bowlofeggs** [comment]
I have cherry-picked the waiverdb portion of this change into #3138.

**Comment by bowlofeggs** [comment]
OK, now I see that the resultdb one does want fedora-messaging too:

```
15:04:46  container.py               495 INFO     Traceback (most recent call last):
15:04:46    File "/usr/bin/resultsdb", line 9, in <module>
15:04:46      load_entry_point('resultsdb==2.2.0', 'console_scripts', 'resultsdb')()
15:04:46    File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 378, in load_entry_point
15:04:46      return get_distribution(dist).load_entry_point(group, name)
15:04:46    File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 2566, in load_entry_point
15:04:46  container.py               495 INFO         return ep.load()
15:04:46    File "/usr/lib/python2.7/site-packages/pkg_resources.py", line 2260, in load
15:04:46  container.py               495 INFO         entry = __import__(self.module_name, globals(),globals(), ['__name__'])
15:04:46    File "/usr/lib/python2.7/site-packages/resultsdb/__init__.py", line 189, in <module>
15:04:46  container.py               495 INFO         from resultsdb.controllers.api_v1 import api as api_v1
15:04:46    File "/usr/lib/python2.7/site-packages/resultsdb/controllers/api_v1.py", line 41, in <module>
15:04:46      from resultsdb.messaging import load_messaging_plugin, create_message, publish_taskotron_message
15:04:46    File "/usr/lib/python2.7/site-packages/resultsdb/messaging.py", line 33, in <module>
15:04:46      from fedora_messaging.api import Message, publish
15:04:46  ImportError: No module named fedora_messaging.api
```

I will cherry pick this entire change into #3138.

**Comment by bowlofeggs** [comment]
This was merged in https://github.com/fedora-infra/bodhi/pull/3138
